test/suites/dev/moonbase/test-storage-growth/test-precompile-storage-growth.ts

```diff
@@ -88,7 +90,7 @@ describeSuite({
           ],
           privateKey: BALTATHAR_PRIVATE_KEY,
           rawTxOnly: true,
-          gas: expectedGas,
+          gas: 91_088n,
```

Comment:
Note: this call fails with OOG if we use a `gas_limit` lower than this number (which is the entire `pov_gas` without refunding), but the actual `gas_used` is the expected one (`54_168n`) and it's correct. 

@RomarQ @ahmadkaouk any clue of what could be happening? I guess that the remaining gas should be refunded to the user after the execution, but not sure why we must indicate `91_088n` here. Instead, a gas limit of `54_168n` should be enough to cover all the transaction cost.

Reply:
At the end this is expected, as we still count for the worst case scenario at te moment of estimating.